ser_scripts/train.py

```diff
@@ -0,0 +1,69 @@
+# train.py
+
+import torch
+from torch import optim
+import torch.nn.functional as F
+import torch.nn as nn
+from ser_scripts.model import model_load
+from ser_scripts.data import create_dataloaders
+
+def train_model(learning_rate, epochs, batch_size):
+
+    class Net(nn.Module):
```

Comment:
this class should be defined once in the model file and imported... not defined in multiple places!!